File: tests/fixtures/dart/project/lib/src/models/item.dart
// Exercises: package: import + show combinator — URI updates, show combinator survives.
import 'package:acme_utils/src/formatter.dart' show Formatter;

class Item {
  final double price;

  const Item({required this.price});

  String formattedPrice() => Formatter().formatValue(price);
}

File: tests/fixtures/go/project/pkg/models/item.go
package models

import "github.com/example/myproject/pkg/utils"

type Item struct {
	Price float64
}

func (i Item) FormattedPrice() string {
	return utils.FormatValue(i.Price)
}

func (i Item) IsPriceValid() bool {
	return utils.IsValid(i.Price)
}
